<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1446</width>
    <height>798</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MeasureTool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,0">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>稳定下限</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lowerValue">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>P/cm3</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>稳定上限</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="upperValue">
            <property name="layoutDirection">
             <enum>Qt::RightToLeft</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>P/cm3</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Cn:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="cnValue"/>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>P/cm3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>平均时间</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="averTime"/>
          </item>
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>s</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>setpoint diameter:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="setMeter"/>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>nm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="startCalibBtn">
            <property name="text">
             <string>开始校准</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_21">
            <property name="text">
             <string>SMPS地址</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="smpsAddr"/>
          </item>
          <item>
           <widget class="QLabel" name="label_22">
            <property name="text">
             <string>CPC地址</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="cpcAddr"/>
          </item>
          <item>
           <widget class="QLabel" name="label_23">
            <property name="text">
             <string>测试设备地址</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="testDevAddr"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="saveBtn">
            <property name="text">
             <string>保存地址</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="0,0,0,0,0">
          <item>
           <widget class="QLabel" name="label_19">
            <property name="text">
             <string>本机地址</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="localAddr"/>
          </item>
          <item>
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>端口</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="listenPort">
            <property name="text">
             <string>11231</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="chartViewLayout"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>U:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_6"/>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>V</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>classifier diameter  (1e-):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="value1e"/>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>nm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>(2e-):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="value2e"/>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>nm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>(3e-):</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="value3e"/>
        </item>
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>nm</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="startBtn">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="menuBtn">
        <property name="text">
         <string>Menu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="tableBtn">
        <property name="text">
         <string>Table</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
